sdk/monitor/query/azmetrics/CHANGELOG.md

```diff
@@ -1,14 +1,13 @@
 # Release History
 
-## 0.1.1 (Unreleased)
+## 1.0.0 (2024-04-09)
 
 ### Features Added
+* First GA
```

Comment:
Eh, I'd just delete this section. This isn't really a feature and the version number expresses the same thing.

Reply:
fixed in [63dda26](https://github.com/Azure/azure-sdk-for-go/pull/22729/commits/63dda26dd58e2aa4a3d108fc27e4cbfa30bfd77c)